### ISSUE TITLE
Let commenters know what happened to the application

### ISSUE THREAD

**Issue by henare** [openaustralia/planningalerts#557]
I was just thinking , "I wonder what happened to that application I commented on?"

It would be great if we could provide the results of an application to commenters so they can see how their comment influenced the application.

**Comment by CloCkWeRX** [comment]
ATDIS feeds do include ```status```; though a lot of scrapers avoid any kind of update to existing applications

**Comment by jamezpolley** [comment]
Duplicate of #653 (which was opened later but has lots more detail). 

Requires #1317

**Comment by mlandauer** [comment]
Yes I think it would be huge if we could let people know whether an application was accepted or rejected after they commented. I would hope that councils do this now but I really have no idea

**Comment by JoannaHill** [comment]
Several more recent and related issues have been opened here: 

https://github.com/openaustralia/planningalerts/issues/1694
https://github.com/openaustralia/planningalerts/issues/1696
https://github.com/openaustralia/planningalerts/issues/1731

I don't think we should close this issue however until all have been reviewed together. Smells like a mini project!
